<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="14" passed="10" failed="3" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-30T23:55:46 IST" name="Suite" finished-at="2023-04-30T23:56:15 IST" duration-ms="29610">
    <groups>
    </groups>
    <test started-at="2023-04-30T23:55:46 IST" name="Test" finished-at="2023-04-30T23:56:15 IST" duration-ms="29610">
      <class name="testEndPoints.AdequateEndpointTests">
        <test-method signature="TestUserRegister()[pri:1, instance:testEndPoints.AdequateEndpointTests@cda4919]" started-at="2023-04-30T23:55:46 IST" name="TestUserRegister" finished-at="2023-04-30T23:55:51 IST" duration-ms="4775" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestUserRegister -->
        <test-method signature="TestValidUserLogin()[pri:2, instance:testEndPoints.AdequateEndpointTests@cda4919]" started-at="2023-04-30T23:55:51 IST" name="TestValidUserLogin" finished-at="2023-04-30T23:55:51 IST" duration-ms="404" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [success] but found [invalid username or password]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [success] but found [invalid username or password]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testEndPoints.AdequateEndpointTests.TestValidUserLogin(AdequateEndpointTests.java:88)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestValidUserLogin -->
        <test-method signature="TestCreateUser()[pri:3, instance:testEndPoints.AdequateEndpointTests@cda4919]" started-at="2023-04-30T23:55:51 IST" name="TestCreateUser" finished-at="2023-04-30T23:55:51 IST" duration-ms="433" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCreateUser -->
        <test-method signature="TestgetUser()[pri:4, instance:testEndPoints.AdequateEndpointTests@cda4919]" started-at="2023-04-30T23:55:51 IST" name="TestgetUser" finished-at="2023-04-30T23:55:52 IST" duration-ms="416" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestgetUser -->
        <test-method signature="TestInValidUserLogin()[pri:5, instance:testEndPoints.AdequateEndpointTests@cda4919]" started-at="2023-04-30T23:55:52 IST" name="TestInValidUserLogin" finished-at="2023-04-30T23:55:52 IST" duration-ms="309" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestInValidUserLogin -->
      </class> <!-- testEndPoints.AdequateEndpointTests -->
      <class name="testEndPoints.MobileStoreEndPointTests">
        <test-method signature="testLoginUser()[pri:1, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:55:46 IST" name="testLoginUser" finished-at="2023-04-30T23:55:50 IST" duration-ms="4617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginUser -->
        <test-method signature="testRegisterUser()[pri:2, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:55:50 IST" name="testRegisterUser" finished-at="2023-04-30T23:55:51 IST" duration-ms="758" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRegisterUser -->
        <test-method signature="testCreateProducts(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:3, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:55:52 IST" name="testCreateProducts" data-provider="Data" finished-at="2023-04-30T23:56:03 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser" duration-ms="10620" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Apple Iphone 13]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[80000]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="org.awaitility.core.ConditionTimeoutException">
            <message>
              <![CDATA[Condition with testEndPoints.MobileStoreEndPointTests was not fulfilled within 10 seconds.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.awaitility.core.ConditionTimeoutException: Condition with testEndPoints.MobileStoreEndPointTests was not fulfilled within 10 seconds.
at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:136)
at org.awaitility.core.CallableCondition.await(CallableCondition.java:79)
at org.awaitility.core.CallableCondition.await(CallableCondition.java:27)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:840)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:802)
at testEndPoints.MobileStoreEndPointTests.testCreateProducts(MobileStoreEndPointTests.java:109)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- org.awaitility.core.ConditionTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateProducts -->
        <test-method signature="testCreateProducts(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:3, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:56:03 IST" name="testCreateProducts" data-provider="Data" finished-at="2023-04-30T23:56:08 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser" duration-ms="5774" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Xiaomi 12 Pro 5G]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[50000]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[150]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateProducts -->
        <test-method signature="testCreateProducts(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:3, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:56:08 IST" name="testCreateProducts" data-provider="Data" finished-at="2023-04-30T23:56:14 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser" duration-ms="5495" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Xiaomi 13 Pro 5G]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[70000]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateProducts -->
        <test-method signature="testGetProduct()[pri:5, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:56:14 IST" name="testGetProduct" finished-at="2023-04-30T23:56:15 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser" duration-ms="577" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetProduct -->
        <test-method signature="testUpdateProduct()[pri:6, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:56:15 IST" name="testUpdateProduct" finished-at="2023-04-30T23:56:15 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser" duration-ms="392" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateProduct -->
        <test-method signature="testDeleteProduct()[pri:7, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:56:15 IST" name="testDeleteProduct" finished-at="2023-04-30T23:56:15 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser" duration-ms="434" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <200> but was <502>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <502>.

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:60)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at testEndPoints.MobileStoreEndPointTests.testDeleteProduct(MobileStoreEndPointTests.java:187)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteProduct -->
        <test-method signature="testGetProducts(java.lang.String)[pri:4, instance:testEndPoints.MobileStoreEndPointTests@33a053d]" started-at="2023-04-30T23:56:15 IST" name="testGetProducts" finished-at="2023-04-30T23:56:15 IST" depends-on-methods="testEndPoints.MobileStoreEndPointTests.testLoginUser, testEndPoints.MobileStoreEndPointTests.testCreateProducts" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method testEndPoints.MobileStoreEndPointTests.testGetProducts() on instance testEndPoints.MobileStoreEndPointTests@33a053d depends on not successfully finished methods 
[testEndPoints.MobileStoreEndPointTests.testLoginUser() on instance testEndPoints.MobileStoreEndPointTests@33a053d
testEndPoints.MobileStoreEndPointTests.testCreateProducts() on instance testEndPoints.MobileStoreEndPointTests@33a053d]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method testEndPoints.MobileStoreEndPointTests.testGetProducts() on instance testEndPoints.MobileStoreEndPointTests@33a053d depends on not successfully finished methods 
[testEndPoints.MobileStoreEndPointTests.testLoginUser() on instance testEndPoints.MobileStoreEndPointTests@33a053d
testEndPoints.MobileStoreEndPointTests.testCreateProducts() on instance testEndPoints.MobileStoreEndPointTests@33a053d]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetProducts -->
      </class> <!-- testEndPoints.MobileStoreEndPointTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
